Start Synthesizing Wav File
SLURM_SUBMIT_DIR = /home/lapis/UGRP/tts/Tacotron2-Wavenet-Korean-TTS
CUDA_HOME = /opt/ohpc/pub/apps/cuda-10.0
CUDA_VISIBLE_DEVICES = 4,5
CUDA_VERSION=10.0
0: n16
0: /home/lapis/UGRP/tts/Tacotron2-Wavenet-Korean-TTS
0: Sat Sep 12 15:48:53 KST 2020
WARNING:tensorflow:From /home/lapis/UGRP/tts/Tacotron2-Wavenet-Korean-TTS/wavenet/ops.py:51: The name tf.layers.Conv2D is deprecated. Please use tf.compat.v1.layers.Conv2D instead.

WARNING:tensorflow:
The TensorFlow contrib module will not be included in TensorFlow 2.0.
For more information, please see:
  * https://github.com/tensorflow/community/blob/master/rfcs/20180907-contrib-sunset.md
  * https://github.com/tensorflow/addons
  * https://github.com/tensorflow/io (for I/O related ops)
If you depend on functionality not listed there, please file an issue.

WARNING:tensorflow:From generate.py:123: The name tf.Session is deprecated. Please use tf.compat.v1.Session instead.

2020-09-12 15:49:06.525123: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcuda.so.1
2020-09-12 15:49:06.814723: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1639] Found device 0 with properties: 
name: GeForce RTX 2080 Ti major: 7 minor: 5 memoryClockRate(GHz): 1.545
pciBusID: 0000:b1:00.0
2020-09-12 15:49:06.817341: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1639] Found device 1 with properties: 
name: GeForce RTX 2080 Ti major: 7 minor: 5 memoryClockRate(GHz): 1.545
pciBusID: 0000:b2:00.0
2020-09-12 15:49:06.819341: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcudart.so.10.0
2020-09-12 15:49:06.825930: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcublas.so.10.0
2020-09-12 15:49:06.835296: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcufft.so.10.0
2020-09-12 15:49:06.853382: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcurand.so.10.0
2020-09-12 15:49:06.875858: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcusolver.so.10.0
2020-09-12 15:49:06.885769: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcusparse.so.10.0
2020-09-12 15:49:06.908798: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcudnn.so.7
2020-09-12 15:49:06.919041: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1767] Adding visible gpu devices: 0, 1
2020-09-12 15:49:06.919539: I tensorflow/core/platform/cpu_feature_guard.cc:142] Your CPU supports instructions that this TensorFlow binary was not compiled to use: AVX2 AVX512F FMA
2020-09-12 15:49:06.933814: I tensorflow/core/platform/profile_utils/cpu_utils.cc:94] CPU Frequency: 2200000000 Hz
2020-09-12 15:49:06.935506: I tensorflow/compiler/xla/service/service.cc:168] XLA service 0x55757b2a63d0 initialized for platform Host (this does not guarantee that XLA will be used). Devices:
2020-09-12 15:49:06.935536: I tensorflow/compiler/xla/service/service.cc:176]   StreamExecutor device (0): Host, Default Version
2020-09-12 15:49:07.342500: I tensorflow/compiler/xla/service/service.cc:168] XLA service 0x55757b309420 initialized for platform CUDA (this does not guarantee that XLA will be used). Devices:
2020-09-12 15:49:07.342590: I tensorflow/compiler/xla/service/service.cc:176]   StreamExecutor device (0): GeForce RTX 2080 Ti, Compute Capability 7.5
2020-09-12 15:49:07.342622: I tensorflow/compiler/xla/service/service.cc:176]   StreamExecutor device (1): GeForce RTX 2080 Ti, Compute Capability 7.5
2020-09-12 15:49:07.346536: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1639] Found device 0 with properties: 
name: GeForce RTX 2080 Ti major: 7 minor: 5 memoryClockRate(GHz): 1.545
pciBusID: 0000:b1:00.0
2020-09-12 15:49:07.349035: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1639] Found device 1 with properties: 
name: GeForce RTX 2080 Ti major: 7 minor: 5 memoryClockRate(GHz): 1.545
pciBusID: 0000:b2:00.0
2020-09-12 15:49:07.349109: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcudart.so.10.0
2020-09-12 15:49:07.349133: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcublas.so.10.0
2020-09-12 15:49:07.349153: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcufft.so.10.0
2020-09-12 15:49:07.349173: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcurand.so.10.0
2020-09-12 15:49:07.349192: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcusolver.so.10.0
2020-09-12 15:49:07.349212: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcusparse.so.10.0
2020-09-12 15:49:07.349232: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcudnn.so.7
2020-09-12 15:49:07.359007: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1767] Adding visible gpu devices: 0, 1
2020-09-12 15:49:07.359076: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcudart.so.10.0
2020-09-12 15:49:07.365614: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1180] Device interconnect StreamExecutor with strength 1 edge matrix:
2020-09-12 15:49:07.365638: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1186]      0 1 
2020-09-12 15:49:07.365652: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1199] 0:   N N 
2020-09-12 15:49:07.365661: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1199] 1:   N N 
2020-09-12 15:49:07.373486: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1325] Created TensorFlow device (/job:localhost/replica:0/task:0/device:GPU:0 with 10283 MB memory) -> physical GPU (device: 0, name: GeForce RTX 2080 Ti, pci bus id: 0000:b1:00.0, compute capability: 7.5)
2020-09-12 15:49:07.377141: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1325] Created TensorFlow device (/job:localhost/replica:0/task:0/device:GPU:1 with 10283 MB memory) -> physical GPU (device: 1, name: GeForce RTX 2080 Ti, pci bus id: 0000:b2:00.0, compute capability: 7.5)
WARNING:tensorflow:From generate.py:145: The name tf.placeholder is deprecated. Please use tf.compat.v1.placeholder instead.

WARNING:tensorflow:From /home/lapis/UGRP/tts/Tacotron2-Wavenet-Korean-TTS/wavenet/model.py:244: The name tf.variable_scope is deprecated. Please use tf.compat.v1.variable_scope instead.

WARNING:tensorflow:From /home/lapis/UGRP/tts/Tacotron2-Wavenet-Korean-TTS/wavenet/model.py:244: The name tf.AUTO_REUSE is deprecated. Please use tf.compat.v1.AUTO_REUSE instead.

WARNING:tensorflow:From /home/lapis/UGRP/tts/Tacotron2-Wavenet-Korean-TTS/wavenet/model.py:217: The name tf.get_variable is deprecated. Please use tf.compat.v1.get_variable instead.

WARNING:tensorflow:From /home/lapis/UGRP/tts/Tacotron2-Wavenet-Korean-TTS/wavenet/model.py:58: The name tf.variables_initializer is deprecated. Please use tf.compat.v1.variables_initializer instead.

WARNING:tensorflow:From /home/lapis/UGRP/tts/Tacotron2-Wavenet-Korean-TTS/wavenet/model.py:44: conv1d (from tensorflow.python.layers.convolutional) is deprecated and will be removed in a future version.
Instructions for updating:
Use `tf.keras.layers.Conv1D` instead.
WARNING:tensorflow:From /home/lapis/.local/lib/python3.7/site-packages/tensorflow_core/python/layers/convolutional.py:218: Layer.apply (from tensorflow.python.keras.engine.base_layer) is deprecated and will be removed in a future version.
Instructions for updating:
Please use `layer.__call__` method instead.
WARNING:tensorflow:From /home/lapis/UGRP/tts/Tacotron2-Wavenet-Korean-TTS/wavenet/model.py:70: The name tf.scatter_update is deprecated. Please use tf.compat.v1.scatter_update instead.

WARNING:tensorflow:From /home/lapis/UGRP/tts/Tacotron2-Wavenet-Korean-TTS/wavenet/model.py:74: dropout (from tensorflow.python.layers.core) is deprecated and will be removed in a future version.
Instructions for updating:
Use keras.layers.dropout instead.
WARNING:tensorflow:From /home/lapis/UGRP/tts/Tacotron2-Wavenet-Korean-TTS/wavenet/model.py:76: The name tf.layers.Conv1D is deprecated. Please use tf.compat.v1.layers.Conv1D instead.

WARNING:tensorflow:From /home/lapis/UGRP/tts/Tacotron2-Wavenet-Korean-TTS/wavenet/mixture.py:103: The name tf.log is deprecated. Please use tf.math.log instead.

WARNING:tensorflow:From /home/lapis/UGRP/tts/Tacotron2-Wavenet-Korean-TTS/wavenet/mixture.py:103: The name tf.random_uniform is deprecated. Please use tf.random.uniform instead.

WARNING:tensorflow:From /home/lapis/UGRP/tts/Tacotron2-Wavenet-Korean-TTS/wavenet/ops.py:65: calling Constant.__init__ (from tensorflow.python.ops.init_ops) with dtype is deprecated and will be removed in a future version.
Instructions for updating:
Call initializer instance with the dtype argument instead of passing it to the constructor
WARNING:tensorflow:From /home/lapis/.local/lib/python3.7/site-packages/tensorflow_core/python/autograph/converters/directives.py:119: The name tf.assert_equal is deprecated. Please use tf.compat.v1.assert_equal instead.

WARNING:tensorflow:From /home/lapis/.local/lib/python3.7/site-packages/tensorflow_core/python/autograph/converters/directives.py:119: The name tf.batch_to_space_nd is deprecated. Please use tf.batch_to_space instead.

WARNING:tensorflow:From generate.py:162: The name tf.global_variables is deprecated. Please use tf.compat.v1.global_variables instead.

WARNING:tensorflow:From generate.py:163: The name tf.train.Saver is deprecated. Please use tf.compat.v1.train.Saver instead.

UPDATE attention_kernel: (31,) -> [31]
UPDATE postnet_kernel_size: (5,) -> [5]
Restoring model from ./logdir-wavenet/train/2020-08-26T00-06-25
Trying to restore saved checkpoints from ./logdir-wavenet/train/2020-08-26T00-06-25 ... No checkpoint found.
Traceback (most recent call last):
  File "/home/lapis/.local/lib/python3.7/site-packages/tensorflow_core/python/client/session.py", line 1365, in _do_call
    return fn(*args)
  File "/home/lapis/.local/lib/python3.7/site-packages/tensorflow_core/python/client/session.py", line 1350, in _run_fn
    target_list, run_metadata)
  File "/home/lapis/.local/lib/python3.7/site-packages/tensorflow_core/python/client/session.py", line 1443, in _call_tf_sessionrun
    run_metadata)
tensorflow.python.framework.errors_impl.FailedPreconditionError: Attempting to use uninitialized value wavenet/SubPixelConvolution_layer_1/bias
	 [[{{node wavenet/SubPixelConvolution_layer_1/bias/read}}]]

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "generate.py", line 280, in <module>
    main()
  File "generate.py", line 206, in main
    upsampled_local_condition_data = sess.run(upsampled_local_condition_data)
  File "/home/lapis/.local/lib/python3.7/site-packages/tensorflow_core/python/client/session.py", line 956, in run
    run_metadata_ptr)
  File "/home/lapis/.local/lib/python3.7/site-packages/tensorflow_core/python/client/session.py", line 1180, in _run
    feed_dict_tensor, options, run_metadata)
  File "/home/lapis/.local/lib/python3.7/site-packages/tensorflow_core/python/client/session.py", line 1359, in _do_run
    run_metadata)
  File "/home/lapis/.local/lib/python3.7/site-packages/tensorflow_core/python/client/session.py", line 1384, in _do_call
    raise type(e)(node_def, op, message)
tensorflow.python.framework.errors_impl.FailedPreconditionError: Attempting to use uninitialized value wavenet/SubPixelConvolution_layer_1/bias
	 [[node wavenet/SubPixelConvolution_layer_1/bias/read (defined at /home/lapis/.local/lib/python3.7/site-packages/tensorflow_core/python/framework/ops.py:1748) ]]

Original stack trace for 'wavenet/SubPixelConvolution_layer_1/bias/read':
  File "generate.py", line 280, in <module>
    main()
  File "generate.py", line 160, in main
    upsampled_local_condition_data = net.create_upsample(mel_input,upsample_type=hparams.upsample_type)
  File "/home/lapis/UGRP/tts/Tacotron2-Wavenet-Korean-TTS/wavenet/model.py", line 135, in create_upsample
    local_condition_batch = subpixel_layer(local_condition_batch)
  File "/home/lapis/.local/lib/python3.7/site-packages/tensorflow_core/python/layers/base.py", line 548, in __call__
    outputs = super(Layer, self).__call__(inputs, *args, **kwargs)
  File "/home/lapis/.local/lib/python3.7/site-packages/tensorflow_core/python/keras/engine/base_layer.py", line 824, in __call__
    self._maybe_build(inputs)
  File "/home/lapis/.local/lib/python3.7/site-packages/tensorflow_core/python/keras/engine/base_layer.py", line 2146, in _maybe_build
    self.build(input_shapes)
  File "/home/lapis/UGRP/tts/Tacotron2-Wavenet-Korean-TTS/wavenet/ops.py", line 94, in build
    super(SubPixelConvolution, self).build(input_shape)
  File "/home/lapis/.local/lib/python3.7/site-packages/tensorflow_core/python/keras/layers/convolutional.py", line 174, in build
    dtype=self.dtype)
  File "/home/lapis/.local/lib/python3.7/site-packages/tensorflow_core/python/layers/base.py", line 461, in add_weight
    **kwargs)
  File "/home/lapis/.local/lib/python3.7/site-packages/tensorflow_core/python/keras/engine/base_layer.py", line 529, in add_weight
    aggregation=aggregation)
  File "/home/lapis/.local/lib/python3.7/site-packages/tensorflow_core/python/training/tracking/base.py", line 712, in _add_variable_with_custom_getter
    **kwargs_for_getter)
  File "/home/lapis/.local/lib/python3.7/site-packages/tensorflow_core/python/ops/variable_scope.py", line 1500, in get_variable
    aggregation=aggregation)
  File "/home/lapis/.local/lib/python3.7/site-packages/tensorflow_core/python/ops/variable_scope.py", line 1243, in get_variable
    aggregation=aggregation)
  File "/home/lapis/.local/lib/python3.7/site-packages/tensorflow_core/python/ops/variable_scope.py", line 567, in get_variable
    aggregation=aggregation)
  File "/home/lapis/.local/lib/python3.7/site-packages/tensorflow_core/python/ops/variable_scope.py", line 519, in _true_getter
    aggregation=aggregation)
  File "/home/lapis/.local/lib/python3.7/site-packages/tensorflow_core/python/ops/variable_scope.py", line 933, in _get_single_variable
    aggregation=aggregation)
  File "/home/lapis/.local/lib/python3.7/site-packages/tensorflow_core/python/ops/variables.py", line 258, in __call__
    return cls._variable_v1_call(*args, **kwargs)
  File "/home/lapis/.local/lib/python3.7/site-packages/tensorflow_core/python/ops/variables.py", line 219, in _variable_v1_call
    shape=shape)
  File "/home/lapis/.local/lib/python3.7/site-packages/tensorflow_core/python/ops/variables.py", line 197, in <lambda>
    previous_getter = lambda **kwargs: default_variable_creator(None, **kwargs)
  File "/home/lapis/.local/lib/python3.7/site-packages/tensorflow_core/python/ops/variable_scope.py", line 2519, in default_variable_creator
    shape=shape)
  File "/home/lapis/.local/lib/python3.7/site-packages/tensorflow_core/python/ops/variables.py", line 262, in __call__
    return super(VariableMetaclass, cls).__call__(*args, **kwargs)
  File "/home/lapis/.local/lib/python3.7/site-packages/tensorflow_core/python/ops/variables.py", line 1688, in __init__
    shape=shape)
  File "/home/lapis/.local/lib/python3.7/site-packages/tensorflow_core/python/ops/variables.py", line 1872, in _init_from_args
    self._snapshot = array_ops.identity(self._variable, name="read")
  File "/home/lapis/.local/lib/python3.7/site-packages/tensorflow_core/python/util/dispatch.py", line 180, in wrapper
    return target(*args, **kwargs)
  File "/home/lapis/.local/lib/python3.7/site-packages/tensorflow_core/python/ops/array_ops.py", line 203, in identity
    ret = gen_array_ops.identity(input, name=name)
  File "/home/lapis/.local/lib/python3.7/site-packages/tensorflow_core/python/ops/gen_array_ops.py", line 4239, in identity
    "Identity", input=input, name=name)
  File "/home/lapis/.local/lib/python3.7/site-packages/tensorflow_core/python/framework/op_def_library.py", line 794, in _apply_op_helper
    op_def=op_def)
  File "/home/lapis/.local/lib/python3.7/site-packages/tensorflow_core/python/util/deprecation.py", line 507, in new_func
    return func(*args, **kwargs)
  File "/home/lapis/.local/lib/python3.7/site-packages/tensorflow_core/python/framework/ops.py", line 3357, in create_op
    attrs, op_def, compute_device)
  File "/home/lapis/.local/lib/python3.7/site-packages/tensorflow_core/python/framework/ops.py", line 3426, in _create_op_internal
    op_def=op_def)
  File "/home/lapis/.local/lib/python3.7/site-packages/tensorflow_core/python/framework/ops.py", line 1748, in __init__
    self._traceback = tf_stack.extract_stack()

Sat Sep 12 15:49:12 KST 2020
             JOBID PARTITION     NAME     USER ST       TIME  NODES NODELIST(REASON)
             29508 gpu-2080t KTacoSyn    lapis  R       0:19      1 n16
##### END #####
